packages/w/wlogout/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : wlogout
+version    : 1.2.2
+release    : 1
+source     :
+    - https://github.com/ArtsyMacaw/wlogout/archive/refs/tags/1.2.2.tar.gz : 4c9204bfa19c73f51176c94c67711f54f3e393301c0809c61ae379054060fa46
+homepage   : https://github.com/ArtsyMacaw/wlogout
+license    : MIT
+component  : desktop
+summary    : A wayland based logout menu
+description: |
+    wlogout is a logout menu for wayland environments
+builddeps  :
```

Comment:
Lets do what the linter said

Reply:
You mean, just use the summary description?